custom-connector-sdk/src/main/java/com/amazonaws/appflow/custom/connector/model/connectorconfiguration/auth/OAuth2Defaults.java

```diff
@@ -47,13 +47,25 @@
 
     /**
      * Token URLs. Required for both OAuth2GrantType - AUTHORIZATION_CODE and CLIENT_CREDENTIALS.
-     * for the cases when token url is required from the AppFlow users please use template url
-     * in the format as : https://{{YOUR_SUBDOMAIN}}.auth.marketingcloudapis.com/v2/token
+     * This URL can be in two format - Static and Template based where we need the input from the Customer to build the complete URL.
+     * For Static URLs you can add the URL directly to this list like 
+     *  - https://mc17nm2rn5qghzw-lk5r70cv2prm.auth.marketingcloudapis.com/v2/token.
```

Comment:
Can you change the Statis Token URL to - https://login.salesforce.com/services/oauth2/token